lib/ansible/plugins/filter/ipaddr.py

```diff
@@ -1059,6 +1059,8 @@ def hwaddr(value, query='', alias='hwaddr'):
 
     try:
         v = netaddr.EUI(value)
+    except netaddr.AddrFormatError:
+        return False
```

Comment:
A new `except` isn't needed.

The bug occurs because the existing `except` doesn't return `False` if not raising another exception.